apps/io-profile/env.example

```diff
@@ -2,17 +2,19 @@ NODE_ENV=dev
 NODE_TLS_REJECT_UNAUTHORIZED=0
 
 FUNCTIONS_WORKER_RUNTIME=node
-WEBSITE_NODE_DEFAULT_VERSION=14.16.0
+WEBSITE_NODE_DEFAULT_VERSION=20.12.2
 APPLICATIONINSIGHTS_CONNECTION_STRING=InstrumentationKey=foo;IngestionEndpoint=http://localhost;LiveEndpoint=http://localhost
 SUBSCRIPTIONS_FEED_TABLE=SubscriptionsFeedByDay
 
-LogsStorageConnection=storageConnectionString
-EventsQueueStorageConnection=storageConnectionString
-IOPSTAPP_STORAGE_CONNECTION_STRING=storageConnectionString
-DURABLE_FUNCTION_STORAGE_CONNECTION_STRING=storageConnectionString
+AzureWebJobsStorage="http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;QueueEndpoint=http://127.0.0.1:10001/devstoreaccount1;TableEndpoint=http://127.0.0.1:10002/devstoreaccount1;"
+LogsStorageConnection="http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;QueueEndpoint=http://127.0.0.1:10001/devstoreaccount1;TableEndpoint=http://127.0.0.1:10002/devstoreaccount1;"
```

Comment:
It seems it's not used anymore

Reply:
you are right. I removed it in https://github.com/pagopa/io-auth-n-identity-domain/pull/256/commits/d3f64f655db0a101c94c85b50bd4ade63d19e896